.circleci/config.yml

```
@@ -501,7 +498,7 @@ jobs:
       - run:
           name: Karma tests
           command: |
-            npm test -- --reporters dots,junit --browsers ChromeHeadless
+            yarn test -- --reporters dots,junit --browsers ChromeHeadless
```

I'm not sure but I think this is not required:
```suggestion
            yarn test --reporters dots,junit --browsers ChromeHeadless
```